src/computer.h

```diff
@@ -116,6 +116,8 @@ class computer
     private:
         // Difficulty of simply logging in
         int security;
+        // Date of next attempt
+        long next_attempt = 0;
```

Comment:
There are 360 turns an hour, 8,640 turns a day, 508,400 turns a cata-year
(3,061,600 a real year).
Compare that with 2^31 ( 2,147,483,648) for a signed int.
I don't think we're in danger of overrunning that one :)